gluten-core/src/main/java/io/glutenproject/substrait/expression/StructLiteralNode.java

```diff
@@ -30,6 +30,9 @@ public StructLiteralNode(InternalRow row, TypeNode typeNode) {
   public LiteralNode getFieldLiteral(int index) {
     InternalRow value = getValue();
     TypeNode type = ((StructNode) getTypeNode()).getFieldTypes().get(index);
+    if (value.isNullAt(index)) {
+      return ExpressionBuilder.makeNullLiteral(type);
```

Comment:
can we add a test case for null value in struct literal ?

Reply:
This change here makes sure null value is not converted to unexpected value, for example 0 for int. Now we still not support null in side struct literal. there are many UT fail with it. Will fix and enable it later.
Thanks.